doc/start/hardware-recommendations.rst

```diff
@@ -143,23 +145,36 @@ write times, as well as total throughput. These physical limitations affect
 overall system performance--especially during recovery. We recommend using a
 dedicated (ideally mirrored) drive for the operating system and software, and
 one drive for each Ceph OSD Daemon you run on the host (modulo NVMe above).
-Many "slow OSD" issues not attributable to hardware failure arise from running
-an operating system and multiple OSDs on the same drive. Since the cost of troubleshooting performance issues on a small cluster likely exceeds the cost of the extra disk drives, you can optimize your cluster design planning by avoiding the temptation to overtax the OSD storage drives.
+Many "slow OSD" issues (when they are not attributable to hardware failure)
+arise from running an operating system and multiple OSDs on the same drive.
+Because it usually costs more to troubleshoot performance issues on a small
+cluster than it costs to buy extra disk drives, optimal cluster design likely
+involves adding more disk drives in order to avoid overtaxing the OSD storage
+drives.
 
```

Comment:
I might elide lines 150-153 (`Because it...`); I think 148-149 convey the idea.

Reply:
I agree with you.

The PR that was the father of this one was intended to be a don't-spill-a-drop rewording of what was already there. But your comment has moved me to spill this drop.